<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HomeScreen</class>
 <widget class="QMainWindow" name="HomeScreen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Home Screen</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>40</y>
      <width>851</width>
      <height>192</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/Login1/logo.png);</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>10</y>
      <width>75</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 10pt &quot;Segoe Print&quot;;</string>
    </property>
    <property name="text">
     <string>LOGOUT</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>230</y>
      <width>248</width>
      <height>36</height>
     </rect>
    </property>
    <property name="whatsThis">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;justify&quot;&gt;aasdf&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 12pt &quot;Segoe Print&quot;;</string>
    </property>
    <property name="text">
     <string>VIEW PARTICIPANT PROFILE</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsView_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>230</y>
      <width>1031</width>
      <height>131</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(165, 185, 255);</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>230</y>
      <width>213</width>
      <height>33</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 12pt &quot;Segoe Script&quot;;</string>
    </property>
    <property name="text">
     <string>REMOVE PARTICIPANT</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>280</y>
      <width>133</width>
      <height>36</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 12pt &quot;Segoe Print&quot;;</string>
    </property>
    <property name="text">
     <string>ADD PROJECT</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_6">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>230</y>
      <width>214</width>
      <height>36</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 12pt &quot;Segoe Print&quot;;</string>
    </property>
    <property name="text">
     <string>REGISTER PARTICIPANT</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_7">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>230</y>
      <width>151</width>
      <height>36</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 12pt &quot;Segoe Print&quot;;</string>
    </property>
    <property name="text">
     <string>MANAGE DUTIES</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_8">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>280</y>
      <width>152</width>
      <height>36</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 12pt &quot;Segoe Print&quot;;</string>
    </property>
    <property name="text">
     <string>CREATE GROUPS</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>280</y>
      <width>157</width>
      <height>36</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 12pt &quot;Segoe Print&quot;;</string>
    </property>
    <property name="text">
     <string>ASSIGN PROJECT</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_9">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>280</y>
      <width>120</width>
      <height>36</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 12pt &quot;Segoe Print&quot;;</string>
    </property>
    <property name="text">
     <string>ADD RESULT</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_10">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>280</y>
      <width>99</width>
      <height>36</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 12pt &quot;Segoe Print&quot;;</string>
    </property>
    <property name="text">
     <string>SCHEDULE</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_11">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>280</y>
      <width>153</width>
      <height>36</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 12pt &quot;Segoe Print&quot;;</string>
    </property>
    <property name="text">
     <string>BUDGET SYSTEM</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_12">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>320</y>
      <width>133</width>
      <height>36</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 12pt &quot;Segoe Print&quot;;</string>
    </property>
    <property name="text">
     <string>SHOW RESULT</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_13">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>320</y>
      <width>133</width>
      <height>36</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 12pt &quot;Segoe Print&quot;;</string>
    </property>
    <property name="text">
     <string>TIMER</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsView_3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1024</width>
      <height>600</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/Login1/Abstract-wallpaper-for-PC-87 (1).jpg);</string>
    </property>
   </widget>
   <zorder>graphicsView_3</zorder>
   <zorder>graphicsView_2</zorder>
   <zorder>graphicsView</zorder>
   <zorder>pushButton_2</zorder>
   <zorder>pushButton_3</zorder>
   <zorder>pushButton_4</zorder>
   <zorder>pushButton_5</zorder>
   <zorder>pushButton_6</zorder>
   <zorder>pushButton_7</zorder>
   <zorder>pushButton_8</zorder>
   <zorder>pushButton</zorder>
   <zorder>pushButton_9</zorder>
   <zorder>pushButton_10</zorder>
   <zorder>pushButton_11</zorder>
   <zorder>pushButton_12</zorder>
   <zorder>pushButton_13</zorder>
  </widget>
 </widget>
 <resources>
  <include location="login1.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>HomeScreen</receiver>
   <slot>logout()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>930</x>
     <y>32</y>
    </hint>
    <hint type="destinationlabel">
     <x>915</x>
     <y>0</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_3</sender>
   <signal>clicked()</signal>
   <receiver>HomeScreen</receiver>
   <slot>watchProfile()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>618</x>
     <y>243</y>
    </hint>
    <hint type="destinationlabel">
     <x>586</x>
     <y>0</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_4</sender>
   <signal>clicked()</signal>
   <receiver>HomeScreen</receiver>
   <slot>goToRemoveParticipant()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>405</x>
     <y>242</y>
    </hint>
    <hint type="destinationlabel">
     <x>366</x>
     <y>0</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_5</sender>
   <signal>clicked()</signal>
   <receiver>HomeScreen</receiver>
   <slot>goToAddProject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>270</x>
     <y>296</y>
    </hint>
    <hint type="destinationlabel">
     <x>314</x>
     <y>0</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_7</sender>
   <signal>clicked()</signal>
   <receiver>HomeScreen</receiver>
   <slot>goToDuties()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>879</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>803</x>
     <y>0</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_6</sender>
   <signal>clicked()</signal>
   <receiver>HomeScreen</receiver>
   <slot>goToRegister()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>135</x>
     <y>243</y>
    </hint>
    <hint type="destinationlabel">
     <x>235</x>
     <y>0</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>HomeScreen</receiver>
   <slot>goToAssProject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>421</x>
     <y>302</y>
    </hint>
    <hint type="destinationlabel">
     <x>417</x>
     <y>0</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_11</sender>
   <signal>clicked()</signal>
   <receiver>HomeScreen</receiver>
   <slot>goToBudget()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>866</x>
     <y>296</y>
    </hint>
    <hint type="destinationlabel">
     <x>755</x>
     <y>0</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_9</sender>
   <signal>clicked()</signal>
   <receiver>HomeScreen</receiver>
   <slot>goToResult()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>599</x>
     <y>301</y>
    </hint>
    <hint type="destinationlabel">
     <x>543</x>
     <y>0</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_12</sender>
   <signal>clicked()</signal>
   <receiver>HomeScreen</receiver>
   <slot>showResult()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>609</x>
     <y>325</y>
    </hint>
    <hint type="destinationlabel">
     <x>733</x>
     <y>0</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_13</sender>
   <signal>clicked()</signal>
   <receiver>HomeScreen</receiver>
   <slot>goToTimer()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>344</x>
     <y>339</y>
    </hint>
    <hint type="destinationlabel">
     <x>670</x>
     <y>0</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_10</sender>
   <signal>clicked()</signal>
   <receiver>HomeScreen</receiver>
   <slot>goTofirstPage()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>739</x>
     <y>288</y>
    </hint>
    <hint type="destinationlabel">
     <x>702</x>
     <y>0</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_8</sender>
   <signal>clicked()</signal>
   <receiver>HomeScreen</receiver>
   <slot>goCreateGroup()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>91</x>
     <y>298</y>
    </hint>
    <hint type="destinationlabel">
     <x>122</x>
     <y>-3</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>logout()</slot>
  <slot>goToProfile()</slot>
  <slot>goToAddParticipant()</slot>
  <slot>watchProfile()</slot>
  <slot>goToRemoveParticipant()</slot>
  <slot>goToAddProject()</slot>
  <slot>goToDuties()</slot>
  <slot>goToRegister()</slot>
  <slot>goToAssProject()</slot>
  <slot>goToBudget()</slot>
  <slot>goToResult()</slot>
  <slot>showResult()</slot>
  <slot>goToTimer()</slot>
  <slot>goTofirstPage()</slot>
  <slot>goCreateGroup()</slot>
 </slots>
</ui>
